planner/core/cacheable_checker.go

```diff
@@ -21,7 +21,12 @@ import (
 
 // Cacheable checks whether the input ast is cacheable.
 func Cacheable(node ast.Node) bool {
-	if _, isSelect := node.(*ast.SelectStmt); !isSelect {
+
```

Comment:
Could we remove this blank line?

Reply:
Thank you for the comment. I'll delete the line from the updated PR.